blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/BlueOceanUrlActionTest.java

```diff
@@ -7,15 +7,14 @@
 import static org.hamcrest.CoreMatchers.instanceOf;
 
 import static org.junit.Assert.assertThat;
-import static org.mockito.Mockito.mock;
 
 public class BlueOceanUrlActionTest {
 
     @Test
     public void testMigration() {
-        BlueOceanUrlObject mock = mock(BlueOceanUrlObject.class);
+        BlueOceanUrlObject urlObject = new BlueOceanUrlObjectImpl(null);
```

Comment:
XStream could not serialize the mock object due to its requirement of reflection for a `java.util.concurrent` type not present in any `Add-Opens` directives. Fixed by removing use of mocks.